hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/payload/SqlTypedRecord.scala

```diff
@@ -17,27 +17,59 @@
 
 package org.apache.spark.sql.hudi.command.payload
 
+import com.google.common.cache.CacheBuilder
```

Comment:
where is this picked up from? We removed guava from deps a while ago.

Reply:
filed
https://issues.apache.org/jira/browse/HUDI-4482 -> remove guava
https://issues.apache.org/jira/browse/HUDI-4483 -> fix checkstyle

@KnightChess would you take these?  i see guava heavily used in procedure, @XuQianJin-Stars maybe you can help review?

Reply:
@xushiyan I will fix it this week